Mathlib/RingTheory/PowerSeries/WellKnown.lean

```diff
@@ -66,6 +67,75 @@ theorem map_invUnitsSub (f : R →+* S) (u : Rˣ) :
 
 end Ring
 
+section invOneSubPow
+
+variable {S : Type*} [CommRing S] (d : ℕ)
+
+/--
+Note that `mk 1` is the constant function `1` so the power series `1 + X + X^2 + ...`.
```

Comment:
Maybe it's just me, I think the doc string is a bit confusing. Maybe the following is better?
```
(1 + X + X^2 + ...) * (1 - X) = 1

Note that the power series `(1 + X + X^2 + ...)` is written as `mk 1` where `1` is the constant function so that `mk 1` is the power series with all coefficient equal to one.
```